picachu/tests/auth/test_auth.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+from flask import url_for
+
+from picachu.config.auth_config import auth_username, auth_password
+
+
+def test_log_in_happy_path(flask_app):
```

Comment:
It is not at all clear what the happy path is. It's a very fuzzy concept that you and I understand, but may not be understood by someone who comes to the project and is not familiar with it. Better to call it something like this: test_log_in_with_the_correct_user_credentials.